pyspark_ai/prompt.py

```diff
@@ -59,6 +59,20 @@
 Observation: OK
 Thought:I now know the final answer.
 Final Answer: SELECT * FROM spark_ai_temp_view_93bcf0 PIVOT (SUM(Amount) FOR Country IN ('USA', 'Canada', 'Mexico', 'China'))"""
+    """QUESTION: Given a Spark temp view `spark_ai_temp_view_19acs2` with the following columns:
+```
+Car STRING
+Color STRING
+Make STRING
+```
+Write a Spark SQL query to retrieve from view `spark_ai_temp_view_19acs2`: If color is gold, what is the total number of cars?
+Thought: I will use the column 'Color' to filter the rows where its value is 'gold' and then select the COUNT(`Car`) 
+because COUNT gives me the total number of cars. I do not use SUM because SUM is not for counting total number.
```

Comment:
can there be a case where we need to use sum for total number?

Reply:
yep let me change this back... was trying to see if I could force it not to use SUM